src/wmnds/components/table/_table.scss

```diff
@@ -39,21 +39,57 @@ $border: #979797;
       display: block;
     }
 
-    tr > th,
-    tr > td {
-      border-bottom: 0;
+    // Hide table headers (but not display: none;, for accessibility)
+    thead tr {
+      position: absolute;
+      top: -9999px;
+      left: -9999px;
     }
 
-    td,
-    th {
-      height: auto;
-      padding: $size-xs 0;
+    tr {
+      padding: $size-md 0;
+      border-bottom: 0;
     }
 
-    tr {
-      padding: $size-xs 0;
-      border-bottom: 1px solid $border;
+    &.wmnds-table--with-header {
```

Comment:
Could the table--with-header just be the default class (.wmnds-table)?
Then a dev only has to add the modifier class if they don't want headers.